### ISSUE TITLE
Question: Redis Expiry Time for keys.

### ISSUE THREAD

**Issue by Agur-A** [ekzhu/datasketch#149]
Hi

First of all thanks for the library!

Is there a way to set an expiry time for the keys in Redis?
I saw this answer https://github.com/ekzhu/datasketch/issues/54, but MongoDB is not an option for me right now.
If anyone has an idea I would love to know  :)

Thanks.

**Comment by ekzhu** [comment]
Good idea. Currently no, but since Redis support expiring keys I am sure it is not too difficult to add it in this package. You are welcome to send a pull request if you like.
